src/pages/collection/series/TmdbLinking.tsx

```diff
@@ -179,7 +179,7 @@ const TmdbLinking = () => {
           TmdbShowID: tmdbId,
           TmdbEpisodeID: overrideId,
           Index: index,
-          Rating: (overrideId === 0 ? MatchRatingType.None : MatchRatingType.UserVerified),
+          Rating: MatchRatingType.UserVerified,
```

Comment:
it's to align the uo while linking to what we will receive from the server after the links are made on the server and the xrefs have been updated.

Reply:
I'm confused by your statement...
Because this change would mean considering unlinked episodes as user overridden.
But when the server returns the new data, it considers the link to not exist at all.. unless that's a bug

Reply:
![image](https://github.com/user-attachments/assets/35a04b2b-a535-4403-8b04-82ed36b7f5eb)
![image](https://github.com/user-attachments/assets/000a6e82-1fa7-47d5-8c45-f1c4f83d9a3a)

eg., I unlinked an episode, it's missing from the xrefs

Reply:
it's missing because you're filtering by tmdb show id. if you remove that restriction/filter then it will show up fine.